Xamarin.Forms.Platform.UAP/Extensions/TextBlockExtensions.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Xamarin.Forms.Platform.UAP.Extensions
+{
+	internal static class TextBlockExtensions
+	{
+		public static void UpdateLineBreakMode(this TextBlock textBlock, LineBreakMode lineBreakMode)
+		{
+			if (textBlock == null)
+				return;
+
+			switch (lineBreakMode)
+			{
+				case LineBreakMode.NoWrap:
+					textBlock.TextTrimming = TextTrimming.Clip;
+					textBlock.TextWrapping = TextWrapping.NoWrap;
+					break;
+				case LineBreakMode.WordWrap:
+					textBlock.TextTrimming = TextTrimming.None;
+					textBlock.TextWrapping = TextWrapping.Wrap;
+					break;
+				case LineBreakMode.CharacterWrap:
+					textBlock.TextTrimming = TextTrimming.WordEllipsis;
+					textBlock.TextWrapping = TextWrapping.Wrap;
+					break;
+				case LineBreakMode.HeadTruncation:
+					// TODO: This truncates at the end.
+					textBlock.TextTrimming = TextTrimming.WordEllipsis;
+					DetermineTruncatedTextWrapping(textBlock);
+					break;
+				case LineBreakMode.TailTruncation:
+					textBlock.TextTrimming = TextTrimming.CharacterEllipsis;
+					DetermineTruncatedTextWrapping(textBlock);
+					break;
+				case LineBreakMode.MiddleTruncation:
+					// TODO: This truncates at the end.
```

Comment:
Nit: Remove TODO

Reply:
@riverar, as you can see here those comments exist before my change, I just moved all the code around. I don't think this is a good idea to remove it.